### PR TITLE
Do not conflate aesthetics and layers

### PR DESCRIPTION
**Pull request by eirianop**
I noticed that the discussion on aesthetics and layers are conflated in the second section of this episode. I refactored the first section and took it slower so that students can really understand the ggplot rules. I have taught this version in a Carpentries training and received feedback from experienced instructors that this was the best presentation of ggplot they have  seen, and I received positive feedback from attendees.

### CODE REVIEW

**Comment by jcoliver** [comment]
Thanks, @eirianop ! We're reviewing this now.

File: _episodes_rmd/08-plot-ggplot2.Rmd [hunk 1]
```diff
@@ -157,18 +209,20 @@ ggplot(data = gapminder, mapping = aes(x=year, y=lifeExp, by=country, color=cont
...
 ```{r lifeExp-layer-example-1}
 ggplot(data = gapminder, mapping = aes(x=year, y=lifeExp, by=country)) +
   geom_line(mapping = aes(color=continent)) + geom_point()
 ```
 
-In this example, the *aesthetic* mapping of **color** has been moved from the
-global plot options in `ggplot` to the `geom_line` layer so it no longer applies
-to the points. Now we can clearly see that the points are drawn on top of the
+
+Notice that we moved `aes(color=continent)` out of the base and into the
```
Comment (jcoliver) [inline _episodes_rmd/08-plot-ggplot2.Rmd:224]: Try changing to "out of the `ggplot` command and into the..."

File: _episodes_rmd/08-plot-ggplot2.Rmd [hunk 2]
```diff
@@ -56,37 +56,87 @@ ggplot(data = gapminder, mapping = aes(x = gdpPercap, y = lifeExp)) +
...
-We've passed in two arguments to `ggplot`. First, we tell `ggplot` what data we
-want to show on our figure, in this example the gapminder data we read in
-earlier. For the second argument,  we passed in the `aes` function, which
-tells `ggplot` how variables in the **data** map to *aesthetic* properties of
-the figure, in this case the **x** and **y** locations. Here we told `ggplot` we
-want to plot the "gdpPercap" column of the gapminder data frame on the x-axis, and
-the "lifeExp" column on the y-axis. Notice that we didn't need to explicitly
-pass `aes` these columns (e.g. `x = gapminder[, "gdpPercap"]`), this is because
-`ggplot` is smart enough to know to look in the **data** for that column!
+It is instructive to break down this line of code piece by piece. *Really*
```
Comment (jcoliver) [inline _episodes_rmd/08-plot-ggplot2.Rmd:59]: I think calling out the two arguments passed in `ggplot` is still warranted. The first four sentences of the original paragraph starting "We've passed in two arguments..." does this well (roughly lines 64-70).

File: _episodes_rmd/08-plot-ggplot2.Rmd [hunk 3]
```diff
@@ -56,37 +56,87 @@ ggplot(data = gapminder, mapping = aes(x = gdpPercap, y = lifeExp)) +
...
+want to plot the "gdpPercap" column of the gapminder data frame on the x-axis, and
+the "lifeExp" column on the y-axis, and color code by continent.
+
+There are many ways to configure aesthetics, and we'll see m ore throughout the hour.
+
+The important thing to know is there are 2 ways to use aesthetics.
+
+1. First way: **Mapping**
+  * This gives us insights about our data. The example above breaks points down by continent
+  * Mapping  configurations are "inherited" by components. We gave `ggplot` aesthetics, and `geom_point` inherited them.
```
Comment (jcoliver) [inline _episodes_rmd/08-plot-ggplot2.Rmd:116]: An additional point in the **Mapping** section could be: 
* We generally pass the names of columns to arguments when we use aesthetics for mapping; this *maps* variables to specific aesthetics of our plot

**Comment by jcoliver** [comment]
Thanks again for this PR, @eirianop . I added a few comments that could be addressed.

File: _episodes_rmd/08-plot-ggplot2.Rmd (hunk 3 above)
Comment (eirianop) [reply _episodes_rmd/08-plot-ggplot2.Rmd:116]: Hi Jeff,

I haven't forgotten about this, I will get to responding to change requests when I have bandwidth. 

Thanks,
Eirian

**Comment by jcoliver** [comment]
Closing this for now, but if there is interest in addressing the requested changes, please feel free to re-open.

**Pull request closed without merging**